src/Service/EventRejectedEmailService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Joindin\Api\Service;
+
+use DateTime;
+
+class EventRejectedEmailService extends BaseEmailService
+{
+    protected $event;
+    protected $website_url;
+
+    public function __construct(array $config, array $recipients, array $event)
```

Comment:
Why would we want to inject 

- a list of recipients
- event information

via constructor?

These arguments should probably be passed to a method that will then actually construct and send a message.

Reply:
I'm not interested in solving the implementation issues of this class in this PR, I'm primarily focused on shipping a feature. I would welcome any code cleanup outside of this scope.